src/Components/Modal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react' 
+import './components.css';
+import CloseButton from './CloseButton';
+import CloseParagraph from './CloseParagraph';
+
+/**
+ * This is a component that return a modal interface
+ */
+
+function Modal({setModalOpen}){
+    const modalBody = `Lorem ipsum dolor sit amet,consectetur adipiscing elit,
```

Comment:
This should be a prop (normally the `children`) otherwise this would not be a reusable component.

Reply:
I don't know if it is the best solution, but in my last commit I am sending it in a property when I call the Modal component inside the App component.